server/app/controllers/admin/AdminProgramController.java

```diff
@@ -151,15 +152,27 @@ public Result create(Request request) {
       return ok(newOneView.render(request, programData, message));
     }
 
-    return getSaveProgramDetailsRedirect(request, result.getResult().id());
+    return getSaveProgramDetailsRedirect(
+        request, result.getResult().id(), /* isInCreationFlow= */ true);
   }
 
   /** Returns an HTML page containing a form to edit a draft program. */
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
   public Result edit(Request request, long id) throws ProgramNotFoundException {
     ProgramDefinition program = programService.getProgramDefinition(id);
     requestChecker.throwIfProgramNotDraft(id);
-    return ok(editView.render(request, program));
+    return ok(editView.render(request, program, ProgramEditStatus.EDIT));
+  }
+
+  /**
+   * Similar to {@link #edit}, but these edits are being performed while an admin is still in the
```

Comment:
question (non-blocking): what do you think about making these the same route and passing in a param for edit vs create?

Reply:
Yeah I'm happy to make it a param instead! Do you think the param should be a boolean or a string representation of the `ProgramEditStatus` enum?

Reply:
yep that makes sense to me!

Reply:
the string representation 👍

Reply:
Updated! I also removed another `isInCreationFlow` boolean and replaced it with the `ProgramEditStatus` enum so everywhere should now be using only that enum.